src/plugins/data/public/ui/search_bar/index.tsx

```diff
@@ -20,6 +18,6 @@ const WrappedSearchBar = (props: SearchBarProps) => (
   </React.Suspense>
 );
 
-export const SearchBar = injectI18n(withKibana(WrappedSearchBar));
+export const SearchBar = WrappedSearchBar;
```

Comment:
Are you sure this is no longer necessary?

Reply:
Thx for catch, I should keep like it was. There was test failure due to this.